sources/packages/web/src/services/http/dto/Institution.dto.ts

```diff
@@ -90,24 +104,54 @@ export interface InstitutionBasicAPIOutDTO {
   hasBusinessGuid: boolean;
 }
 
+export interface CreateInstitution {
+  operatingName: string;
+  primaryPhone: string;
+  primaryEmail: string;
+  website: string;
+  regulatingBody: string;
+  otherRegulatingBody?: string;
+  establishedDate: string;
+  primaryContactFirstName: string;
+  primaryContactLastName: string;
+  primaryContactEmail: string;
+  primaryContactPhone: string;
+  institutionType: number;
+  mailingAddress: AddressDetailsFormAPIDTO;
+}
+
 /**
  * DTO for institution creation by the institution user during the on board process
  * when the institution profile and the admin user must be created altogether.
  */
-export interface CreateInstitutionAPIInDTO {
+export class CreateInstitutionAPIInDTO implements CreateInstitution {
```

Comment:
Just to clarify, will the properties of CreateInstitution be considered by plainToClass as they are not decorated?

Reply:
The properties from the interface `CreateInstitution` are implemented in the class `CreateInstitutionAPIInDTO` and properly decorated.
![image](https://github.com/bcgov/SIMS/assets/61259237/2c106e0a-fb47-4a7d-88b9-a7ba157001d8)

Reply:
If `CreateInstitution` is declared as a class and added `@expose` to it's properties we can just extend it right(without having to re-declare it's properties)?

Let me know if I am missing something.

Reply:
Thanks for the change.